site2/docs/security-overview.md

```diff
@@ -4,28 +4,23 @@ title: Pulsar Security Overview
 sidebar_label: Overview
 ---
 
-Apache Pulsar is the central message bus for a business. It is frequently used to store mission-critical data, and therefore enabling security features are crucial.
+As the central message bus for a business, Apache Pulsar is frequently used for storing mission-critical data. Therefore, enabling security features in Pulsar is crucial.
 
-By default, there is no encryption, authentication, or authorization configured. Any client can communicate to Apache Pulsar via plain text service urls.
-It is critical that access via these plain text service urls is restricted to trusted clients only. Network segmentation and/or authorization ACLs can be used
-to restrict access to trusted IPs in such cases. If neither is used, the cluster is wide open and can be accessed by anyone.
+By default, Pulsar configures no encryption, authentication, or authorization. Any client can communicate to Apache Pulsar via plain text service urls. So we must ensure that Pulsar accessing via these plain text service urls is restricted to trusted clients only. In such cases, you can use Network segmentation and/or authorization ACLs to restrict access to trusted IPs. If you use neither, the state of cluster is wide open and anyone can access the cluster.
```

Comment:
I'm unsure the meaning of "there is no encryption, authentication, or authorization configured" equals to "Pulsar configures no encryption, authentication, or authorization", you can ask help from Pulsar developers.

Reply:
seems to be fine to me.

Reply:
@Monica-zy   Use "URLs" instead of "urls". It's proper noun.

Reply:
ok

Reply:
OK, already update this comment.